dashboard/utils.py

```diff
@@ -298,6 +298,19 @@ def get_all_final_grades(self):
         )
         return {grade.course_run.edx_course_key: grade for grade in grades}
 
+    def get_passing_final_grades_for_course(self, course):
+        """
```

Comment:
please add a test where user has more the one grade and it returns best grade first.